src/components/tools/Charts/Histogram_copy/Annotation.svelte

```diff
@@ -0,0 +1,49 @@
+<script>
```

Comment:
Is there a reason there's a duplication of the `Historgram` chart? Might be worth adding a comment about it unless you're planning on de-dupping them soon.

Reply:
Will remove copy. It's not used.